compiler/tests-common-new/tests/org/jetbrains/kotlin/test/frontend/classic/handlers/FirTestDataConsistencyHandler.kt

```diff
@@ -28,32 +26,17 @@ class FirTestDataConsistencyHandler(testServices: TestServices) : AfterAnalysisC
         if (testData.extension == "kts") return
         if (FirDiagnosticsDirectives.FIR_IDENTICAL in moduleStructure.allDirectives) return
         val firTestData = testData.firTestDataFile
-        if (!firTestData.exists()) {
```

Comment:
Please don't remove this code. It is needed to provide comfortable test writing for guys who are working on FE 1.0 and write test specially for it. It's very annoying to copy-paste testdata and run corresponding test manually

Reply:
The problem is that it doesn't work... the first thing that FIR tests do is copy the FE1.0 test data, which may or may not match what FIR actually outputs (and can easily be done manually anyway) - and since the file now "exists" for purposes of comparions, you just get an error, which you need to manually fix. So far the only convenient way to regenerate FIR test data that I've found is to hack JUnit5Assertions.kt to rewrite the expected file. I wanted to implement *proper* regeneration, but couldn't do it with the way tests work with files, since you can't have different values for "source file" and "expected file".

Reply:
It kinda works in JPS actually, because with it test runner in IDEA can show multiple diff windows for multiple `FileComparasionFailures` in same test, so it's possible to apply changes from `.kt` file to `.fir.kt` at a first test run. So please, don't remove this functionality.

You can vote for request to support multiple diffs in tests which are ran with gradle (https://youtrack.jetbrains.com/issue/IDEA-253597)

Reply:
I reverted this change